src/portable/raspberrypi/rp2040/dcd_rp2040.c

```diff
@@ -64,7 +64,7 @@ static struct hw_endpoint *hw_endpoint_get_by_addr(uint8_t ep_addr)
 }
 static void _hw_endpoint_alloc(struct hw_endpoint *ep)
 {
-    uint size = TU_MIN(64, ep->wMaxPacketSize);
+    uint16_t size = TU_MIN(64, ep->wMaxPacketSize);
```

Comment:
TU_MIN() -> tu_min16() would be better, inline function is better than macro (although the stack uses tons of macro already :smile: :smile: )